package.json

```diff
@@ -36,5 +52,69 @@
     "webpack-cli": "^4.5.0",
     "webpack-dev-server": "^3.11.2",
     "webpack-merge": "^5.7.3"
+  },
+  "husky": {
+    "hooks": {
+      "pre-commit": "lint-staged"
+    }
+  },
+  "lint-staged": {
+    "src/**/*.{js, jsx}": [
+      "eslint",
+      "prettier --write"
+    ]
+  },
+  "babel": {
+    "presets": [
+      "@babel/preset-env",
+      "@babel/preset-react"
+    ]
+  },
+  "eslintConfig": {
```

Comment:
Let's move `eslint` config outside the `package.json`
please create an `.eslintrc` config file

Reply:
Updated in 0b91c07